examples/config/operator7.example.yaml

```diff
@@ -2,9 +2,9 @@ privKey: /data/operator7/encrypted_private_key.json
 privKeyPassword: /data/operator7/password
 port: 3030
 operatorID: 7
+storeShare: false
```

Comment:
so this PR means we don't store shares anymore right? so this option in the config should be deprecated I guess, in config files and the node code..

Reply:
yes, this is a leftover. Removed